File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Category.java
package com.adaptionsoft.games.uglytrivia;

import java.util.LinkedList;

public class Category {
    private final String name;
    private final LinkedList<String> questions = new LinkedList<>();

    public Category(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public LinkedList<String> getQuestions() {
        return questions;
    }

    public String askQuestion() {
        return questions.removeFirst();
    }

    public void addQuestion(int num) {
        questions.addLast(String.format("%s Question %d", name, num));
    }
}

File: exercises/csharp/TriviaGame/TriviaGame/Category.cs
using System.Collections.Generic;

namespace TriviaGame
{
    public class Category
    {
        public string name { get; set; }
        public LinkedList<string> questions { get; set; }

        public Category(string name, LinkedList<string> questions)
        {
            this.name = name;
            this.questions = questions;
        }

        private string CreateQuestionText(int index)
        {
            return $"{name} Question " + index;
        }

        public void AddQuestion(int questionIndex)
        {
            questions.AddLast(CreateQuestionText(questionIndex));
        }
    }
}
